k8s/aat/common/da/da-cos-api.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: helm.fluxcd.io/v1
```

Comment:
this must be kustomized, please follow https://github.com/hmcts/cnp-flux-config/blob/master/docs/app-deployment.md

Reply:
Customisation done as per fluxV2 steps inside apps folder.

Reply:
still not done